resources/language/resource.language.sv_se/strings.po

```diff
@@ -198,16 +198,16 @@ msgstr "Aktivera DEMO-läge"
 
 msgctxt "#30805"
 msgid "Show an UpNext pop-up…"
-msgstr "Visa en popup för UpNext…"
+msgstr "Visa en UpNext-popup…"
```

Comment:
Any reason this is `Visa en UpNext-popup…` and not `Visa en UpNext popup…` or `Visa en UpNext pop-up…`?

Reply:
We really don't have any nice translation for pop-up in the swedish language and when i changed it to `UpNext-popup` i can't use two hyphen, so i removed it (you can see that the old translation for pop-up was popup).

When i rearranged the words i need to add a hyphen between the words because UpNext isn't a Swedish word.

If both should have been swedish words i should have spelled it `UpNextpopup` as one word.